library/bignum_mod_raw.c

```diff
@@ -182,6 +182,18 @@ int mbedtls_mpi_mod_raw_from_mont_rep( mbedtls_mpi_uint *X,
     mbedtls_free( T );
     return( 0 );
 }
+
+void mbedtls_mpi_mod_raw_neg( mbedtls_mpi_uint *X,
+                              const mbedtls_mpi_uint *A,
+                              const mbedtls_mpi_mod_modulus *m )
```

Comment:
Should be called `N` to follow the naming convention.

Reply:
The `m` notation is used in both mod and mod_raw, and the convention's ticket is still under design review. We should not block this pr until we decide either way. 

If a change from `N` to `m` across both modules would be required, a follow up issue will be raised and added to the technical debt.